news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: post
+title: "Community Mentorship Program Fall 2017 Signups now Open"
+permalink: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+date: 2017-07-17 20:45:00 +0000
+tumblr_url: http://osunews.tumblr.com/post/163075746433/community-mentorship-program-fall-2017-signups-now-open
+---
+
+Have you ever thought about getting more involved in the mapping community, but have no idea where to begin? The Community Mentorship Program may be the place for you!
+
+![Community Mentorship Program](wiki/shared/news/2017-07-17-community-mentorship-program-fall-2017-signups-now-open/banner2.jpg)
+
+Since the beginning of the mentorship program in July of 2016, the program has gone through four cycles, each lasting approximately 10 weeks. Aspiring new mappers have been taught the fundamentals of mapping from experienced mappers in the community, and some have even gone on to rank their own beatmaps!
```

Comment:
add a little tl;dr after this with something like

"If you're interested in learning to mod and map from some of osu!'s best and brightest, this is the place for you!"